dhall-toml/dhall-toml.cabal

```
@@ -0,0 +1,78 @@
+Name:               dhall-toml
+Version:            0.1.0
+Cabal-Version:      >=1.10
+Build-Type:         Simple
+Tested-With:        GHC == 8.4.3, GHC == 8.6.1
+License:            BSD3
+License-File:       LICENSE
+Copyright:          2021 ear7h
+Author:             ear7h
+Maintainer:         Gabriel439@gmail.com
+Bug-Reports:        https://github.com/dhall-lang/dhall-haskell/issues
+Synopsis:           Convert between Dhall and TOML
+Description:
+    Use this package if you want to convert between dhall expressions and TOML.
+    You can use this package as a library or an executable:
+    .
+    * See "Dhall.TOML" or "Dhall.TOMLToDhall" modules if you want to use this
+      package as a library
+    * Use @dhall-to-toml@, @toml-to-dhall@ programs from this package if you
+      want an executable.
+    .
+    The "Dhall.TOML" and "Dhall.TOMLToDhall" modules also contains instructions
+    for how to use this package
+Category:           Compiler
+Extra-Source-Files:
+    README.md
+    CHANGELOG.md
+    tasty/data/*.dhall
+    tasty/data/*.toml
+
+Source-Repository head
+    Type: git
+    Location: https://github.com/dhall-lang/dhall-haskell/tree/master/dhall-toml
+
+Library
+    Hs-Source-Dirs:   src
+    Build-Depends:
+        base  >= 4.14   && < 4.15 ,
+        dhall >= 1.39.0 && < 1.40
+    Exposed-Modules:
+        Dhall.TOML
```

IMHO this is the more convential and preferable way of handling abbreviations in CamelCase: It's more readable and easier to type:

```suggestion
        Dhall.Toml
```